common/transport/amqp/src/amqp.ts

```diff
@@ -145,44 +235,190 @@ export class Amqp {
           putToken: (audience, token, callback) => callback(new errors.NotConnectedError()),
         },
         connecting: {
-          _onEnter: (config, connectCallback) => {
-            let connectError = null;
-            const connectErrorHandler = (err) => {
-              connectError = err;
-            };
-
-            this._amqp.on(_amqpClientError, connectErrorHandler);
-            this._amqp.connect(config.uri, config.policyOverride)
-              .then((result) => {
-                debug('AMQP transport connected.');
-                this._amqp.on(_amqpClientError, amqpErrorHandler);
-                this._amqp.removeListener(_amqpClientError, connectErrorHandler);
-                this._fsm.transition('connected', connectCallback, result);
-                return null;
-              })
-              .catch((err) => {
-                this._amqp.removeListener(_amqpClientError, connectErrorHandler);
-                /*Codes_SRS_NODE_COMMON_AMQP_16_003: [The `connect` method shall call the `done` callback if the connection fails.] */
-                this._fsm.transition('disconnected', connectCallback, connectError || err);
-              });
+          _onEnter: (connectionParameters, connectCallback) => {
+            this._rheaContainer.options.sender_options = {properties: {'com.microsoft:client-version': this._config.userAgentString}, reconnect: false};
```

Comment:
>{properties: {'com.microsoft:client-version': this._config.userAgentString}, reconnect: false}; [](start = 57, length = 95)

nit: please make those multi-lines for readability

Reply:
Done

---
In reply to: [209131170](https://github.com/Azure/azure-iot-sdk-node/pull/344#discussion_r209131170) [](ancestors = 209131170)